openshift-client/src/main/java/io/fabric8/openshift/client/dsl/Processable.java

```diff
@@ -13,13 +13,31 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
 package io.fabric8.openshift.client.dsl;
 
+import java.io.File;
+import java.io.InputStream;
+import java.util.Map;
+
 import io.fabric8.openshift.client.ParameterValue;
 
 public interface Processable<T> {
 
+
+  T process(File f);
+
+  T process(InputStream is);
+
+  T process(Map<String, String> map);
+
   T process(ParameterValue... values);
 
+  T processLocally(File f);
```

Comment:
Can you add comments describing difference between `process` and `processLocally` please?